src/runtime_src/core/pcie/driver/linux/xocl/subdev/ulite.c

```diff
@@ -438,8 +436,6 @@ static struct uart_driver xcl_ulite_driver = {
 	.owner		= THIS_MODULE,
 	.driver_name	= XOCL_DEVNAME(XOCL_UARTLITE),
 	.dev_name	= ULITE_NAME,
-	.major		= ULITE_MAJOR,
-	.minor		= ULITE_MINOR,
 	.nr		= ULITE_NR_UARTS,

```

Comment:
This means that if you do not specify Major, then linux uart will allocate one, correct?

Reply:
exactly,

__register_chrdev_region(unsigned int major, unsigned int baseminor,
			   int minorct, const char *name)
{
....

	if (major == 0) {
		ret = find_dynamic_major();